src/main/java/io/vertx/gsoc2018/qotd/QuoteOfTheDayVerticle.java

```diff
@@ -3,50 +3,38 @@
 import io.vertx.core.AbstractVerticle;
 import io.vertx.core.Future;
 import io.vertx.core.json.JsonObject;
-import io.vertx.ext.jdbc.JDBCClient;
-import io.vertx.ext.sql.SQLConnection;
 
 /**
  * @author Thomas Segismont
+ * @author Billy Yuan
  */
 public class QuoteOfTheDayVerticle extends AbstractVerticle {
 
-  private JDBCClient jdbcClient;
-
   @Override
   public void start(Future<Void> startFuture) throws Exception {
     JsonObject jdbcConfig = new JsonObject()
-        .put("url", "jdbc:h2:mem:test;DATABASE_TO_UPPER=false;DB_CLOSE_DELAY=-1")
-        .put("driver_class", "org.h2.Driver");
+      .put("url", "jdbc:h2:mem:test;DATABASE_TO_UPPER=false;DB_CLOSE_DELAY=-1")
+      .put("driver_class", "org.h2.Driver");
 
-    jdbcClient = JDBCClient.createShared(vertx, jdbcConfig);
+    DatabaseService databaseService = new DatabaseServiceImpl(vertx, jdbcConfig);
 
-    Future<Void> initSchema = runScript("classpath:db.sql");
-    Future<Void> importData = runScript("classpath:import.sql");
+    Future<Void> prepareDatabaseFuture = Future.future();
+    databaseService.prepareDatabase(prepareDatabaseFuture);
 
     int port = config().getInteger("http.port", 8080);
 
     // TODO setup database and web server and eventually...
-    startFuture.complete();
-  }
-
-  private Future<Void> runScript(String script) {
-    Future<Void> future = Future.future();
-    jdbcClient.getConnection(getConn -> {
-      if (getConn.succeeded()) {
-        SQLConnection connection = getConn.result();
-        connection.execute("RUNSCRIPT FROM '" + script + "'", exec -> {
```

Comment:
Why not keep the `RUNSCRIPT` command from H2? Works like a charm and no need to manually load the SQL

Reply:
I did not know the RUNSCRIPT command before, done in https://github.com/tsegismont/vertx-gsoc-2018-qotd/pull/4/commits/db64ea673452da95ccb2cf6070a1a95a6424371d.

Reply:
:+1: